### PR TITLE
Feature/improve performance

### PR DESCRIPTION
**Pull request by Quiwin**
Replace use of offset with `WHERE ... ORDER BY` on primary key. On one of our database with millions of large rows, this improved performance by more than 4(went from 2h+ to 30 minutes).
See https://use-the-index-luke.com/sql/partial-results/fetch-next-page

Add --check-columns option to restrict columns to check for equality, this did not improve performance in our case but can still be useful.

### CODE REVIEW

File: pgdatadiff/main.py [hunk 1]
```diff
@@ -12,6 +12,7 @@
   --only-sequences   Only compare seqences, exclude data
   --count-only       Do a quick test based on counts alone
   --chunk-size=10000       The chunk size when comparing data [default: 10000]
+  --check-columns=column-name    Restrict check to given columns, repeatable. Primary key is always checked.
```
Comment (teckwan) [inline pgdatadiff/main.py]:
```suggestion
  --check-columns=column-name    Restrict check to given columns. Repeat to add multiple columns. Primary key is always checked.
```

File: pgdatadiff/pgdatadiff.py [hunk 2]
```diff
@@ -19,7 +19,14 @@ def make_session(connection_string):
...
 class DBDiff(object):
 
-    def __init__(self, firstdb, seconddb, chunk_size=10000, count_only=False):
+    def __init__(
+        self,
+        firstdb,
+        seconddb,
+        chunk_size=10000,
+        count_only=False,
+        check_columns=None
```
Comment (teckwan) [inline pgdatadiff/pgdatadiff.py:28]: The default case is not handled if I'm not wrong? Do you check all the columns when value is `None`?

Comment (Quiwin) [reply pgdatadiff/pgdatadiff.py:28]: Yes, this is done here https://github.com/klaxit/pgdatadiff/blob/6802dccec9d5a7652e31b7651309c97b0b9990c9/pgdatadiff/pgdatadiff.py#L93

**Review by teckwan — APPROVED** [review]
LGTM!

**Pull request merged**